android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -835,6 +823,10 @@ private void finishPlayerInitialization() {
         setControls(controls);
         applyModifiers();
         startBufferCheckTimer();
+
+        if (player != null && youboraPlugin == null && (analyticsMeta != null && !analyticsMeta.getBoolean("contentIsLive") && contentId != analyticsMeta.getString("contentId"))) {
```

Comment:
reason we need `contentIsLive` check is because for VOD the `setAnalyticsMeta` gets called before player is initialised which breaks youbora, player needs to be initialised before initialising youbora.